book.toml

```
@@ -54,3 +54,5 @@ warning-policy = "error"
 "/diagnostics/sessiondiagnostic.html" = "diagnostic-structs.html"
 "/diagnostics/diagnostic-codes.html" = "error-codes.html"
 "/miri.html" = "const-eval/interpret.html"
+"/tests/integration.html" = "tests/ecosystem.html"
+"/tests/headers.html" = "tests/directives.html"
```

These need to be relative to the source directory.

```suggestion
"/tests/integration.html" = "ecosystem.html"
"/tests/headers.html" = "directives.html"
```

---

Wait wdym relative to the source directory? Because e.g. directives.html is still under the `tests/` folder no?

---

I changed the right-side to be as suggested, though I can't claim I understand this redirect syntax exactly 🤷

---

Ah okay thanks for the explanation.